### PR TITLE
Add `calculateBiggestFiles` typing

### PR DESCRIPTION
**Pull request by costinsin**
## Type of change

-   [x] 🧑‍💻 Improvement

## Description

The `calculateBiggestFiles` function was having `any` types for it's parameters. That introduced some bugs because the expected typing was not the actual typing in other contexts. This change fixes it.

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Comment by costinsin** [comment]
Current dependencies on/for this PR:
* dev
  * **PR #561** <a href="https://app.graphite.dev/github/pr/Genez-io/genezio/561?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/Genez-io/genezio/561?utm_source=stack-comment).

**Review by andreia-oca — APPROVED** [review]
Looks good to me, but I don't have an in-depth view of this part of the codebase.
@Tudor-Ang has the ownership here, so please wait for his review before merging.

**Pull request merged**
